nuxeo-features/nuxeo-admin-center/nuxeo-admin-center-jsf/src/main/java/org/nuxeo/connect/client/jsf/ConnectStatusActionBean.java

```diff
@@ -132,8 +132,8 @@ protected ConnectRegistrationService getService() {
     }
 
     @Factory(scope = ScopeType.APPLICATION, value = "registredConnectInstance")
-    public boolean isRegistred() {
-        return getService().isInstanceRegistred();
+    public boolean isRegistered() {
```

Comment:
Will be handled in PR https://github.com/nuxeo/nuxeo/pull/891

Should I keep the previous method to keep the compatibility? (adding a `deprecated` instead of removing it)